auth.php

```diff
@@ -93,6 +95,9 @@ public function canDo($cap) {
         global $ACT;
         #      print_r($cap);
         if(is_null($this->chained_auth)) {
+            if ($cap == "external") {
+                return $this->any_external;
```

Comment:
This is where I see the problems with mixing "trustexternal" and "regular" auth plugins... Dokuwiki won't call any of the regular methods as soon as this returns "true". See comment on "trustexternal" method below.